src/core/features/auto-exposure-roi-feature.h

```diff
@@ -0,0 +1,21 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <core/features/feature-interface.h>
+
+#include <string>
+
+
+namespace librealsense {
+
+class auto_exposure_roi_feature : public feature_interface
+{
+public:
+    auto_exposure_roi_feature() : feature_interface( "Auto exposure ROI feature" )
```

Comment:
Should `feature` really be part of the name?

Reply:
Some of the features are implemented using options. I did not want to confuse between the two.

Reply:
Why would they be confused? When you ask if a feature is supported, it doesn't look in the options, does it?

Reply:
Might also want to think about related features. For example, "auto-exposure" (should it be capitalized? is it case-sensitive?) is required for "auto-exposure-roi". Should it be called `auto-exposure/roi`?